cloudinary-image-management-and-manipulation-in-the-cloud-cdn/cloudinary.php

```diff
@@ -288,49 +299,83 @@ function fix_url($url, $post_id) {
     return $url;
   }
 
+  /**
+   * Build Cloudinary URL for resized image
+   *
+   * @param string         $url - original image url
+   * @param array          $metadata - original image metadata
+   * @param string|array   $size - target size. Can be array with width, height parameters, or
+   *                               string with predefined sizes, see $this->get_wp_sizes for available values
+   *
+   * @return false|array Array containing the image URL, width, height, and boolean for whether
+   *                     the image is an intermediate size. False on failure.
+   */
   function build_resize_url($url, $metadata, $size) {
-    if (preg_match('#(.*?)/(v[0-9]+/.*)$#', $url, $matches)) {
-      if (!$size) {
-        return array($url, $metadata["width"], $metadata["height"], true);
-      }
-      if (is_array($size)) {
-        $wanted = array("width" => $size[0], "height" => $size[1]);
-        $crop = false;
-      } else {
-        $sizes = $this->get_wp_sizes();
-        $wanted = $sizes[$size];
-        $crop = $wanted["crop"];
-      }
-      $transformation = "";
-      $src_w = $dst_w = $metadata["width"];
-      $src_h = $dst_h = $metadata["height"];
-      if ($crop) {
-        $resized = image_resize_dimensions($metadata['width'], $metadata['height'], $wanted['width'], $wanted['height'], true);
-        if ($resized) {
-          list ($dst_x, $dst_y, $src_x, $src_y, $dst_w, $dst_h, $src_w, $src_h) = $resized;
-          $transformation = "c_crop,h_$src_h,w_$src_w,x_$src_x,y_$src_y/";
+    // Check if this is a Cloudinary URL
+    if (!preg_match('#(.*?)/(v[0-9]+/.*)$#', $url, $matches)) {
+      return false;
+    }
+
+    if (!$size) {
+      return array($url, $metadata["width"], $metadata["height"], false);
```

Comment:
@konforti , looks like the last argument(boolean for whether the image is an intermediate size) should be false, we had true. Can you please confirm?